packages/realm-server/tests/card-endpoints-test.ts

```diff
@@ -99,70 +105,14 @@ module(basename(__filename), function () {
             fileSystem,
           }));
 
-          request = supertest(testRealmHttpServer);
+          setTestRequest(supertest(testRealmHttpServer));
```

Comment:
This callback formation is needed so `setupMatrixRoom` later in the file can be extracted as a test helper, it relies on this `request` but that’s not set until the `beforeEach` has executed.

`setupPermissionedRealm` is repeated in 13 files, I‘ll open a followup PR to extract it, with named properties as it’s getting to be That Time